installer/setup_base.txt

```diff
@@ -18,63 +18,82 @@ const config = {
     "domain": {{domain}},
     "port": {{port}}
   },
+  
+  // Intents the bot needs.
+  // By default GuideBot needs Guilds, Guild Messages and Direct Messages to work.
+  // For join messages to work you need Guild Members, which is privileged and requires extra setup.
+  // For more info about intents see the README.
+  intents: {{intents}},
 
   // PERMISSION LEVEL DEFINITIONS.
   permLevels: [
-    {
-      level: 0,
+    // This is the lowest permisison level, this is for non-roled users.
```

Comment:
You can still have a role and still be level 0, please remove the mention of `non-roled users`

Reply:
ok